### ISSUE TITLE
False positive on using `MAX` of signed integer

### ISSUE THREAD

**Issue by bluebear94** [intellij-rust/intellij-rust#10228]
## Environment

* **IntelliJ Rust plugin version:** 0.4.185.5086-223
* **Rust toolchain version:** 1.70.0-nightly (7820b62d2 2023-03-05) x86_64-unknown-linux-gnu
* **IDE name and version:** IntelliJ IDEA 2022.3.1 Community Edition (IC-223.8214.52)
* **Operating system:** Linux 6.1.8-arch1-1
* **Macro expansion:** enabled

## Problem description

The `MAX` constants of built-in signed integer types are falsely detected as private.

## Steps to reproduce

```rust
fn foo() {
    let _ = i16::MAX; // also i8, i32, etc.
}
```

gives a ‘Constant `MAX` is private [E0603]’ error.

**Comment by ik1ne** [comment]
Same here, my environment:
* **IntelliJ Rust plugin version:** 0.4.192.5324-231
* **Rust toolchain version:** 1.69.0 (84c898d65 2023-04-16) aarch64-apple-darwin
* **IDE name and version:** CLion 2023.1.1 (CL-231.8109.222)
* **Operating system:** macOS 13.3.1
* **Macro expansion:** enabled
* **Additional experimental features:** org.rust.macros.proc.attr, org.rust.macros.proc

**Comment by Maksim20023** [comment]
Hi! I'm closing this ticket because the issue is fixed (please see  #10688). This fix is available from the Rust plugin 0.4.199.5414 version and higher. Please ensure that your installed plugin is [up-to-date](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#update). If this issue still reproduces for you, please reopen the ticket.
